runtime/heiko/src/lib.rs

```diff
@@ -676,7 +676,7 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    pub const TransactionByteFee: Balance = 1 * MILLICENTS;
```

Comment:
will be good to keep `1` as we may want to change it to `2` or something else later, that will be simpler